units/meson.build

```diff
@@ -260,9 +260,9 @@ in_units = [
          'sysinit.target.wants/ initrd-root-fs.target.wants/'],
         ['user-runtime-dir@.service',            ''],
         ['user@.service',                        ''],
-        ['systemd-pcrphase-initrd.service',      'HAVE_GNU_EFI ENABLE_INITRD',
+        ['systemd-pcrphase-initrd.service',      'HAVE_OPENSSL HAVE_TPM2 ENABLE_INITRD',
```

Comment:
should also use HAVE_GNU_EFI as condition, since we don't operate on PCR 11 unless a UKI with systemd-stub was used